src/doc/guide.md

```diff
@@ -570,11 +562,10 @@ if x == 5i {
 }
 ```
 
-If we changed the value of `x` to something else, this line would not print.
-More specifically, if the expression after the `if` evaluates to `true`, then
-the block is executed. If it's `false`, then it is not.
+As we might expect, the block is executed when the `if` guard expression
```

Comment:
`x == 5i` is a guard expression, which is what I was refering to. Should I clarify?